packages/dds/sequence/README.md

```diff
@@ -170,3 +170,9 @@ Just like any other sequence, the Sparse Matrix is made of segments. The segment
     },
 ]
 ```
+
+# SharedObjectSequence and SharedNumberSequence
+
+SharedObjectSequence and SharedNumberSequence are very similar distributed data structures. The only difference is the type of content they support. SharedNumberSequence only supports numbers as content, where SharedObjectSequence supports any JSON serializable object. Both DDSes support inserting, removing, and annotating content. Each piece of content -- that is, each number or object -- will occupy a single position in the sequence. The length of the sequence is the count of content items in the sequence.
+
+ An important note is that, unlike an array, positions are not guaranteed remain constant. The positions will increase or decrease as content is added or removed from the sequence. To track or pass a reference to a specific piece of content within the sequence you should find its segment via `segment = s.getContainingSegment(position)` and then use `pos = s.getPosition(segment)` to get its current position in the tree.
```

Comment:
I do not think about positions as "will increase or decrease".
I think better way to say - position of an element will change as content is added or removed.
"increase or decrease" wording feels like describes a property that I can change (i.e. it sort of implies there is "shift down" and "shift up" operations available to me).